README.md

```diff
@@ -597,12 +597,12 @@
     });
 
     // good
-    [1, 2, 3].map((x) => {
-      return x * x;
-    });
+    [1, 2, 3].map(x => x * x);
```

Comment:
Let's keep the original example here...or let's update the example function to be more than one-line.

We try to introduce one-rule by example at a time prevent confusion. In this example, we have the arrow function, the fits on one line, and the only one argument rules being applied.